backend/fms_core/tests/test_resources.py

```diff
@@ -70,6 +70,10 @@ def setUp(self) -> None:
         self.rr = ContainerRenameResource()
         self.pr = ProjectLinkSampleResource()
 
+    def create_projects(self):
+        self.project1, _ = Project.objects.get_or_create(name="TestProject1", status="Ongoing")
```

Comment:
I think we should have a default status for projects in order to prevent having to initialize them everywhere to a hard coded identical string.

Reply:
You mean modifying the attribute of "status" in Project model by adding a default value of "Ongoing(Open)"

Reply:
yeah i think most of the time we'll create Open Projects (is Ongoing or Open the right status?)...

Reply:
Yes, I agree that most of the times the user will create a Project it will have an Open status

Reply:
If Open is the correct status don't forget to fix that on the receiving end PR if you want to use ongoing here ...

Reply:
Agree. Default should be Ongoing.

Reply:
So, Open/Close are the correct options. However, the active PR's are not updated to these options (since they were merged a few days ago), we would have to rebase.